src/js/components/Image.js

```diff
@@ -60,7 +60,7 @@ class Image extends React.Component {
       <img
         className={classes}
         onError={this.onImageError}
-        {...Util.omit(props, ["className"])}
+        {...Util.omit(props, ["className", "fallbackSrc"])}
```

Comment:
It seems like we would only ever want to pass the `src` prop, so why not do `src={props.src}` directly?